### PR TITLE
Html g UI branch

### PR DESCRIPTION
**Pull request by StefanHaboeck**

### CODE REVIEW

File: emba.sh [hunk 1]
```diff
@@ -63,6 +63,8 @@ main()
   export SHORT_PATH=0           # short paths in cli output
   export ONLY_DEP=0             # test only dependency
   export FORCE=0
+  export HTML=0
+  export GHIDRA=1
```
Comment (m-1-k-3) [inline emba.sh:67]: please remove the Ghidra stuff from here

File: emba.sh [hunk 2]
```diff
@@ -71,6 +73,9 @@ main()
   export MOD_DIR="./modules"
   export VUL_FEED_DB="$EXT_DIR""/allitems.csv"
   export BASE_LINUX_FILES="$CONFIG_DIR""/linux_common_files.txt"
+  export HTML_PATH="./html/html-files"         # directory for html path
```
Comment (m-1-k-3) [inline emba.sh:76]: Do we really need this directory? Could we move this to $CONFIG_DIR?

File: emba.sh [hunk 3]
```diff
@@ -110,6 +117,9 @@ main()
       F)
         export FORCE=1
         ;;
+      g)
```
Comment (m-1-k-3) [inline emba.sh:120]: remove ghidra

File: modules/S05_firmware_details.sh [hunk 4]
```diff
@@ -10,7 +10,7 @@
 #
 # emba is licensed under GPLv3
 #
-# Author(s): Michael Messner, Pascal Eckmann
+# Author(s): Michael Messner, Pascal Eckmann, Stefan Haböck
```
Comment (m-1-k-3) [inline modules/S05_firmware_details.sh:13]: Contributors: Stefan Haboeck

File: modules/S100_command_inj_check.sh [hunk 5]
```diff
@@ -10,7 +10,7 @@
 #
 # emba is licensed under GPLv3
 #
-# Author(s): Michael Messner, Pascal Eckmann
+# Author(s): Michael Messner, Pascal Eckmann, Stefan Haböck
```
Comment (m-1-k-3) [inline modules/S100_command_inj_check.sh:13]: Contributors: Stefan Haboeck

File: modules/S100_command_inj_check.sh [hunk 6]
```diff
@@ -52,8 +53,13 @@ S100_command_inj_check()
         done
       fi
     done
+    CONTENT_AVAILABLE=1
   else
     print_output "[-] No directories or files used for web scripts found"
   fi
+  
+  if [[ $HTML == 1 ]]; then
```
Comment (m-1-k-3) [inline modules/S100_command_inj_check.sh:61]: could we move this out of every module to the caller?

File: modules/S105_deep_key_search.sh [hunk 7]
```diff
@@ -10,7 +10,7 @@
 #
 # emba is licensed under GPLv3
 #
-# Author(s): Michael Messner, Pascal Eckmann
+# Author(s): Michael Messner, Pascal Eckmann, Stefan Haböck
```
Comment (m-1-k-3) [inline modules/S105_deep_key_search.sh:13]: Contributors: Stefan Haboeck

File: emba.sh (hunk 2 above)
Comment (p4cx) [reply emba.sh:76]: best variant would be: `/config/html/style.css`

**Comment by m-1-k-3** [comment]
You are currently running into multiple conflicts. Could you please load the current master into your branch and then proceed ...

Thank you

**Comment by m-1-k-3** [comment]
This one is replaced via https://github.com/e-m-b-a/emba/pull/35

I will close this pull request now

**Pull request closed without merging**